tests/unittest/_torch/modules/test_fused_moe.py

```diff
@@ -34,7 +34,7 @@
 
 @pytest.mark.parametrize(
     "moe_cls, dtype, experts, RoutingMethodCls",
-    product([FusedMoE, VanillaMoE], [torch.float16, torch.bfloat16],
+    product([CutlassFusedMoE, VanillaMoE], [torch.float16, torch.bfloat16],
```

Comment:
Any chance we could make some unit tests for TRTLLM backend, I see we only have E2E model tests which are much harder to debug

Reply:
I'll add it in a follow up PR.